package.json

```diff
@@ -88,7 +93,14 @@
     "stylelint-scss": "3.5.4",
     "stylelint-webpack-plugin": "0.10.5",
     "supertest": "3.4.2",
+    "ts-loader": "5.4.5",
+    "tslint": "5.16.0",
+    "tslint-config-prettier": "1.18.0",
+    "tslint-plugin-prettier": "2.0.1",
+    "tslint-react": "4.0.0",
     "typeface-roboto": "0.0.54",
+    "typescript": "3.4.5",
```

Comment:
For linting issues, we must use as higher version. `"typescript": "3.2.1"`. This might change in the future.